tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -654,6 +663,15 @@ where
 				}
 				Ok(())
 			},
+			TrustedCall::set_identity_networks(signer, who, identity, web3networks) => {
+				debug!("set_identity_networks, networks: {:?}", web3networks);
+				// only support DI requests from the signer but we leave the room for changes
+				ensure!(signer == who, Self::Error::Dispatch("Unauthorized signer".to_string()));
```

Comment:
We can have another function `ensure_self`

Reply:
Sure